services/identity-service/src/vc/vc.service.spec.ts

```diff
@@ -38,19 +38,19 @@ describe('DidService', () => {
     const signedPayload = await service.sign(signingDID, 'Hello!');
     expect(signedPayload).toBeDefined();
     expect(signedPayload.publicKey).toBeDefined();
-    expect(signedPayload.signed).toBeDefined();
+    expect(signedPayload.jws).toBeDefined();
   });
 
   it('should verify a signed payload successfully', async () => {
     const signedPayload = await service.sign(signingDID, 'Hello!');
-    const verified = await service.verify(signingDID, signedPayload.signed);
+    const verified = await service.verify(signingDID, signedPayload.jws);
```

Comment:
Run the test cases and see if they are all working.

Reply:
all the tests are working which were working before